Lexplorer/Pages/AccountDetails.razor

```diff
@@ -194,6 +240,7 @@
             cts = localCTS;
             try
             {
+                currentPage = int.TryParse(nftPageNumber, out int dummy) ? dummy : 1;
```

Comment:
Why the separate currentPage? gotoNFTPage is read-write and returns the same.

Reply:
Because I use it to sync the selected page in pager. If I use gotoNFTPage it will end up in an infinite loop.

Reply:
This is if you are on a small screen, to sync the two pagers, and also if you go directly to a url with nftPageNumber parameter set. It's not great, but it works and doesn't go into a loop

Reply:
It took me a while, but I think I found a solution without duplicating currentPage. Please have a look at my take on this in the latest commit.

Reply:
Works like a charm! NICE! :)